docker/release/nanoserver/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# escape=`
+FROM microsoft/nanoserver
+MAINTAINER brycem@microsoft.com
+LABEL Readme.md="https://github.com/PowerShell/PowerShell/blob/master/docker/README.md"
+LABEL Description="This Dockerfile will install and build the latest reslease of PS on NanoServer."
+
+ARG POWERSHELL_ZIP=https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.10/powershell-6.0.0-alpha.10-win10-x64.zip
```

Comment:
These two ARGs would need to be edited when the next release comes out.  Do you need a Pester test that'd break if it doesn't match?